Sharpmake/Project.Configuration.cs

```diff
@@ -2220,6 +2220,12 @@ public enum ProjectFilter
                 /// This is what we tell the build system we're going to produce.
                 /// </summary>
                 public string Output = "";
+
+                /// <summary>
+                /// Optional string to hint to the build system at what to treat the output from the build command as
```

Comment:
This should probably have a comment similar to other fields:
```
/// Not supported by FASTBuild.
```

Reply:
Commend updated